### PR TITLE
The essdee and polo gym vest ipd submission errors are handled

### PR DESCRIPTION
**Pull request by KaviyaPeriyasamy**

### CODE REVIEW

File: apparelo/apparelo/doctype/item_production_detail/item_production_detail.json [hunk 1]
```diff
@@ -93,7 +93,6 @@
    "options": "Combined IPDs"
   },
   {
-   "depends_on": "eval: doc.docstatus == 1",
    "fieldname": "administrator_section_section",
```
Comment (Vigneshsekar) [inline apparelo/apparelo/doctype/item_production_detail/item_production_detail.json]:
```suggestion
   "depends_on": "eval: doc.docstatus == 1",
   "fieldname": "administrator_section_section",
```

**Review by Vigneshsekar — CHANGES_REQUESTED** [review]

File: apparelo/apparelo/doctype/item_production_detail/item_production_detail.py [hunk 2]
```diff
@@ -223,7 +226,7 @@ def create_process_details(self):
 				if process.input_item:
 					pass
 				if process.ipd_name:
-    					if process.input_index:
+					if process.input_index:
```
Comment (Vigneshsekar) [inline apparelo/apparelo/doctype/item_production_detail/item_production_detail.py:229]: You did some indentation change here, yesterday, Is this conflicting or removing that?

**Review by karthikeyan5 — APPROVED** [review]
LGTM! :stuck_out_tongue:

**Comment by karthikeyan5** [comment]
@KaviyaPeriyasamy please don't mention the specific product names... it is more informative to mention the actual problem that was found...

**Pull request merged**